bundle/src/main/java/com/adobe/acs/commons/wcm/properties/shared/impl/SharedValueMapResourceAdapterImpl.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2016 Adobe
```

Comment:
nit: copyrights on new files should have current year

Reply:
they're not copyrights. they're copypastes.